npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fxa-content-server",
-  "version": "1.132.1",
```

Comment:
`npm-shrinkwrap.json` should not be updated unless you are also installing new dependencies. The recommended way of installing dependencies is by running `npm ci` rather than `npm install`, the former does not modify this file whereas the latter does.